db.json

```diff
@@ -0,0 +1,39 @@
+{
+  "videos": [
```

Comment:
I'm still thinking that it shouldn't be in the repository. You can make a test and add it in the testing block :D